sectionsUtilityTool/src/main/webapp/assets/js/utils.js

```diff
@@ -195,9 +195,9 @@ $(document).on('click', '.postCourseNameChange', function (e) {
     url: url
     }).done(function( msg ) {
      $('.courseTitleTextContainer').hide();
-      reportSuccess('Course <strong>' + $thisCourseCode.text() + '</strong> renamed to <strong>' + msg.course_code) + '</strong>';
+      reportSuccess('Course <strong>' + $thisCourseCode.text() + '</strong> renamed to <strong>' + msg.course_code + '</strong>');
       $thisCourseCode.text(msg.course_code);
-      $thisCourseName.text(msg.name)
+      $thisCourseName.text(msg.name);
```

Comment:
It is clearer to have the semicolons even if not strictly required..

On Wed, Mar 4, 2015 at 1:51 PM, Zhen Qian <notifications@github.com> wrote:

> In sectionsUtilityTool/src/main/webapp/assets/js/utils.js
> <https://github.com/tl-its-umich-edu/canvas/pull/17#discussion_r25801410>:
>
> >        $thisCourseCode.text(msg.course_code);
> > -      $thisCourseName.text(msg.name)
> > +      $thisCourseName.text(msg.name);
>
> I thought the semicolon character is not required on the last line
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tl-its-umich-edu/canvas/pull/17/files#r25801410>.
>